.github/workflows/ci.yml

```diff
@@ -20,14 +19,30 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.8
+      - name: Set up conda
+        run: |
+          # $CONDA is an environment variable pointing to the root of the miniconda directory
+          $CONDA/bin/conda env update --file environment.yml --name base
       - name: Install Python dependencies
         run: |
+          eval "$(conda shell.bash hook)"
+          conda activate base
           python -m pip install --upgrade pip
           make install requirements
           pip install -r docs/requirements.txt
-      - name: Build and test Sphinx docs
-        run: make docs
+      - name: Run the quilc & qvm Docker images
+        run: |
+          eval "$(conda shell.bash hook)"
+          conda activate base
+          docker run --rm -idt -p 5000:5000 rigetti/qvm -S
+          docker run --rm -idt -p 5555:5555 rigetti/quilc -R
```

Comment:
What's the purpose of this?

Reply:
I was just trying stuff to avoid the doctests with pyquil fail. I will check this is not the case – the solution may have come from elsewhere.